src/routes/index.js

```diff
@@ -14,7 +14,7 @@ router.get('/items', getItems)
 router.get('/items/:id', getItem)
 router.post('/items', authMid, postItem)
 router.put('/items/:id', editItem)
-
+router.delete('/items/:id', deleteItem)
```

Comment:
```suggestion
router.delete('/items/:id', authMid, deleteItem)
```
Oh! I forget to mention this. We should allow only the user who have valid token.
This change should work.
Just make sure this works to delete an item.